sdk/log/logger.go

```diff
@@ -50,14 +50,24 @@ func (l *logger) Emit(ctx context.Context, r log.Record) {
 // processed, true will be returned by default. A value of false will only be
 // returned if it can be positively verified that no Processor will process.
 func (l *logger) Enabled(ctx context.Context, param log.EnabledParameters) bool {
+	sc := trace.SpanContextFromContext(ctx)
```

Comment:
I am not sure if we should do that. Calling this always introduces an overhead. In needed a processor can always do it on its own as it has this information in the context. I think we can remove the 3 span context fields from EnabledPararamters.

Reply:
https://github.com/open-telemetry/opentelemetry-go/pull/6271/commits/4df66696a9469176f4baeda3d53d90c383ec71df